cypress/integration/journalistCanCreateArticles.feature.js

```diff
@@ -1,96 +1,98 @@
 /* eslint-disable no-undef */
-describe("Journalist can create articles", () => {
+describe('Journalist can create articles', () => {
   beforeEach(() => {
     cy.server();
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/auth/sign_in",
-      response: "fixture:journalist_can_login.json",
+      method: 'POST',
+      url: 'http://localhost:3000/api/auth/sign_in',
+      response: 'fixture:journalist_can_login.json',
       headers: {
-        uid: "journalist@mail.com",
+        uid: 'journalist@mail.com',
       },
     });
     cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/auth/validate_token**",
-      response: "fixture:journalist_can_login.json",
+      method: 'GET',
+      url: 'http://localhost:3000/api/auth/validate_token**',
+      response: 'fixture:journalist_can_login.json',
     });
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/articles",
-      response: { message: "Your article was successfully created!" },
+      method: 'POST',
+      url: 'http://localhost:3000/api/articles',
+      response: { message: 'Your article was successfully created!' },
     });
-    cy.visit("/");
+    cy.visit('/');
   });
 
-  it("successfully", () => {
+  it('successfully', () => {
+    const title = 'Covid vaccine is found';
     cy.get("[data-cy='login-form']").within(() => {
-      cy.get("[data-cy='email']").type("journalist@mail.com");
-      cy.get("[data-cy='password']").type("password");
-      cy.get("[data-cy='submit-btn']").contains("Submit").click();
+      cy.get("[data-cy='email']").type('journalist@mail.com');
+      cy.get("[data-cy='password']").type('password');
+      cy.get("[data-cy='submit-btn']").contains('Submit').click();
     });
     cy.get('[data-cy="create-article-form"]').within(() => {
-      cy.get('[data-cy="input-title"]').type("Covid vaccine is found");
+      cy.get('[data-cy="input-title"]').type(title);
       cy.get('[data-cy="input-sub-title"]').type(
-        "Each person should receive two doses of vaccine"
+        'Each person should receive two doses of vaccine'
       );
       cy.get('[data-cy="input-content"]').type(
-        "Russia has distributed vaccines months ago"
+        'Russia has distributed vaccines months ago'
       );
+      cy.get('[name="file_input"]').attachFile('image.png');
```

Comment:
If the image is not there or broken, should it not be an alternative text there?

Reply:
We’ll consider this- we felt that the image preview takes care of this, and I believe that unlike the other form fields, adding an image was meant to be an option for journalists rather than a requirement.

Reply:
Removed the constants from the test